crates/vhost/src/vhost_user/slave_req.rs

```diff
@@ -65,44 +67,44 @@ impl SlaveFsCacheReqInternal {
     }
 }
 
-/// Request proxy to send vhost-user-fs slave requests to the master through the slave
+/// Request proxy to send vhost-user slave requests to the master through the slave
 /// communication channel.
 ///
-/// The [SlaveFsCacheReq] acts as a message proxy to forward vhost-user-fs slave requests to the
+/// The [Slave] acts as a message proxy to forward vhost-user slave requests to the
 /// master through the vhost-user slave communication channel. The forwarded messages will be
 /// handled by the [MasterReqHandler] server.
 ///
-/// [SlaveFsCacheReq]: struct.SlaveFsCacheReq.html
+/// [Slave]: struct.Slave.html
 /// [MasterReqHandler]: struct.MasterReqHandler.html
 #[derive(Clone)]
-pub struct SlaveFsCacheReq {
+pub struct Slave {
     // underlying Unix domain socket for communication
-    node: Arc<Mutex<SlaveFsCacheReqInternal>>,
+    node: Arc<Mutex<SlaveInternal>>,
 }
 
-impl SlaveFsCacheReq {
+impl Slave {
```

Comment:
Is `Slave` a bit generic?

Maybe `SlaveProxy` or `SlaveReqProxy` (I'm not good with names :-)

(i'd like one day to rename Master/Slave to frontend/backend, but I don't think it is for this PR)

Reply:
This is how `Slave` and `Master` look after this change:

```
pub struct Slave {
    // underlying Unix domain socket for communication
    node: Arc<Mutex<SlaveInternal>>,
}

pub struct Master {
    node: Arc<Mutex<MasterInternal>>,
}
```

And so I went with just `Slave` here.

Reply:
Got it, good for now!